hack/build-clusteradm-image.sh

```diff
@@ -72,12 +73,7 @@ clusteradm_version=${clusteradm_version#v}
 echo "Using clusteradm v${clusteradm_version}."
 
 git clone -b "v$clusteradm_version" --depth 1 https://github.com/open-cluster-management-io/clusteradm.git "$clusteradm_folder"
```

Comment:
This command would get a bit simpler if it came after the `cd "$clusteradm_folder"`.